ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -285,6 +286,9 @@ object BigSampler extends Command {
                                         keyFn: T => U,
                                         maxKeySize: Int = 1e6.toInt,
                                         byteEncoding: ByteEncoding = RawEncoding)
+                                      (implicit coder0: Coder[T],
```

Comment:
`U: Coder` and `T: Coder` bounds instead. Potentially can remove the ClassTag bounds now but not certain